prov/rxm/src/rxm_cq.c

```diff
@@ -700,13 +707,7 @@ static ssize_t rxm_rndv_send_ack_inject(struct rxm_rx_buf *rx_buf)
 	};
 
 	assert(rx_buf->conn);
-
-	pkt.hdr.op		= ofi_op_msg;
-	pkt.hdr.version		= OFI_OP_VERSION;
-	pkt.ctrl_hdr.version	= RXM_CTRL_VERSION;
-	pkt.ctrl_hdr.type	= rxm_ctrl_rndv_ack;
-	pkt.ctrl_hdr.conn_id 	= rx_buf->conn->handle.remote_key;
-	pkt.ctrl_hdr.msg_id 	= rx_buf->pkt.ctrl_hdr.msg_id;
+	pkt.ctrl_hdr.conn_id = rx_buf->conn->handle.remote_key;
```

Comment:
This is initialized above.

Reply:
@shefty Thanks. I deleted assert() call in rxm_rndv_send_ack_inject(), because on 716 line assert() is already called.